client/packages/system/src/Stock/Components/StockLineEditModal.tsx

```diff
@@ -61,10 +67,10 @@ export const StockLineEditModal: FC<StockLineEditModalProps> = ({
     message: t('messages.confirm-save-stock-line'),
   });
 
-  if (!stockLine) return null;
-
   const { draft, onUpdate, onSave } = useDraftStockLine(stockLine);
 
+  if (!stockLine || !draft) return null;
```

Comment:
This one was also tricky. I changed useDraftStockLine to be able to take an undefined stockLine. In this case it returns an undefined draft.